### ISSUE TITLE
it_table_shapes_test.py:41-44: After the form.save()...

### ISSUE THREAD

**Issue by 0pdd** [timelesslounge/timelessis#286]
The puzzle `206-144cbf13` from #206 has to be resolved: 

https://github.com/timelesslounge/timelessis/blob/3c79e5aed367090b3493f764ba6916d48660094a/tests/integration/it_table_shapes_test.py#L41-L44

The puzzle was created by Krzysztof Krason on 18-Feb-19. 

  

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0crat** [comment]
@emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa) please, pay attention to this issue

**Comment by 0crat** [comment]
Job [#286](https://github.com/timelesslounge/timelessis/issues/286) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
The job [#286](https://github.com/timelesslounge/timelessis/issues/286) assigned to @krzyk[/z](https://www.0crat.com/u/krzyk), here is [why](https://www.0crat.com/footprint/CF9LG1YCA/005f31a0-8d5f-4e64-9b73-80c3ae106f85); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be a monetary reward for this job

**Comment by krzyk** [comment]
@0crat refuse

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/timelesslounge/timelessis/issues/286#issuecomment-465023030))

@krzyk The user @krzyk[/z](https://www.0crat.com/u/krzyk) resigned from [#286](https://github.com/timelesslounge/timelessis/issues/286), please stop working. Reason for job resignation: Order was cancelled

**Comment by 0crat** [comment]
Tasks refusal is discouraged, see [§6](http://www.zerocracy.com/policy.html#6): -15 point(s) just awarded to @krzyk[/z](https://www.0crat.com/u/krzyk)

**Comment by 0crat** [comment]
The job [#286](https://github.com/timelesslounge/timelessis/issues/286) assigned to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil), here is [why](https://www.0crat.com/footprint/CF9LG1YCA/eb05b212-6c8d-4e63-9274-c443116a58fc); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be a monetary reward for this job

**Comment by 0crat** [comment]
@amihaiemil[/z](https://www.0crat.com/u/amihaiemil) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by amihaiemil** [comment]
@0pdd waiting for https://github.com/timelesslounge/timelessis/issues/415 . I started on this ticket, but it's a bit confusing. Some puzzle got move without being solved and another older puzzle (which had been solved), got back in (this [one](https://github.com/timelesslounge/timelessis/issues/205))

**Comment by 0pdd** [comment]
> @0pdd waiting for https://github.com/timelesslounge/timelessis/issues/415 . I started on this ticket...

  I see you're talking to me, but I can't reply since I'm not a chat bot.

**Comment by amihaiemil** [comment]
@0crat waiting for https://github.com/timelesslounge/timelessis/issues/415 . I started on this ticket, but it's a bit confusing. Some puzzle got removed without being solved and another older puzzle (which had been solved), got back in (this [one](https://github.com/timelesslounge/timelessis/issues/205))

**Comment by 0crat** [comment]
> @0crat waiting for https://github.com/timelesslounge/timelessis/issues/415 . I started on this ti... ([here](https://github.com/timelesslounge/timelessis/issues/286#issuecomment-467904886))

@amihaiemil The impediment for [#286](https://github.com/timelesslounge/timelessis/issues/286) was registered successfully by @amihaiemil[/z](https://www.0crat.com/u/amihaiemil)

**Comment by emilianodellacasa** [comment]
@amihaiemil I think there has been some mess during a branch merge. I will try to open again issue #415 and at the same time I would kindly ask you to delete puzzle for #205. Do you think it would work that way?

**Comment by amihaiemil** [comment]
@emilianodellacasa yes sure. I wanted to do something in the same direction as well :) I'll take up on it as soon as possible.

**Comment by amihaiemil** [comment]
@emilianodellacasa the puzzles were switched in PR #384, [here](https://github.com/timelesslounge/timelessis/pull/384/files#diff-69fdbdff8d7c8e99f10595a5215e8cf2L37). My bad, I didn't notice...

**Comment by emilianodellacasa** [comment]
@amihaiemil Is the situation clear now? Do you have more impediments on this issue?

**Comment by zverovich** [comment]
@0crat refuse

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/timelesslounge/timelessis/issues/286#issuecomment-485643646))

@zverovich The user @amihaiemil[/z](https://www.0crat.com/u/amihaiemil) resigned from [#286](https://github.com/timelesslounge/timelessis/issues/286), please stop working. Reason for job resignation: Order was cancelled
